cityofaustin/atd-vz-data
# Add a non-coa roadways layer to the database

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/15302

## Testing
**URL to test:** 
Locally

**Steps to test:**
1. Check out this branch, start your local db, run the migrations. 
2. You can just use the geojson i already created from running the first script and just test running the second script to insert the data into the db. Follow steps in the README and run
`node insert_roadways.js local`
4. Inspect the data in the table
5. Run the down migrations

---
#### Ship list
